config/routes.yaml

```diff
@@ -0,0 +1,5 @@
+content:
+    path: /content/{id}
+    controller: Libero\ContentPageBundle\Controller\ContentController
```

Comment:
Currently allows for one instance, and the route has to be set manually. Will need to make bundle configuration to set up multiple.